pandas/tests/extension/test_numpy.py

```diff
@@ -28,6 +32,31 @@ def dtype(request):
     return PandasDtype(np.dtype(request.param))
 
 
+orig_setitem = pd.core.internals.Block.setitem
```

Comment:
can you use monkeypatch instead?

Reply:
this does use monkeypatch.  the monkeypatched method calls the original method